Makefile.am

```diff
@@ -2,3 +2,25 @@
 
 EXTRA_DIST = autogen.sh
 SUBDIRS = src include docs visualc_net
+
+CLEANFILES = \
```

Comment:
Maybe this can be put in a separate commit and included with other PR to be merged earlier?

Reply:
Yes; it's unrelated cleanup of the build system.